README.md

```diff
@@ -146,6 +146,14 @@ sudo apt-get install    \
 **Note:** LLDB currently requires at least `swig-1.3.40` but will successfully build
 with version 2 shipped with Ubuntu.
 
+In addition, you'll need to set up Git Large File Storage for your user account:
+
+```
+curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
```

Comment:
This makes me *really* nervous. We should absolutely not be asking users to execute random scripts with superuser privileges in the README.

Reply:
This is the official installation instruction provided by the git-lfs team: https://github.com/git-lfs/git-lfs/wiki/Installation#debian-and-ubuntu

Alternatively, the README can just link to the page, instead of copying the instructions here?